kde-base/marble/marble-18.08.1.recipe

```diff
@@ -10,9 +10,9 @@ The user interface is clean, simple and easy to use."
 HOMEPAGE="https://marble.kde.org/"
 COPYRIGHT="2007-2018 by the authors of Marble Virtual Globe"
```

Comment:
```diff
-COPYRIGHT="2007-2018 by the authors of Marble Virtual Globe"
+COPYRIGHT="2007-2018 Marble Virtual Globe authors"
```

This is just a suggestion, since the copyright string is shorter this way.